### ISSUE TITLE
Content gone in 1.9.2 but works in 1.9.1

### ISSUE THREAD

**Issue by StephanSchuster** [wirecube/android_additive_animations#19]
I just upgraded from 1.7.5 to 1.9.2 and most of my app's content was not shown anymore. In lots of places I fade in/out content with AdditiveAnimator. After two hours of searching for the root cause I switched from 1.9.2 back to 1.9.1 and suddenly everything is working again.

Might there be a bug in 1.9.2? I saw that lots of visibility/fade related framework code has been changed in 1.9.2.

This is a typical pattern which works in 1.9.1 but does NOT work in 1.9.2:

```
AdditiveAnimator.animate(contentViews)
    .setDuration(UI.CARDS_INTRO_DURATION)
    .setInterpolator(new DecelerateInterpolator())
    .fadeVisibility(show ? View.VISIBLE : View.GONE)
    .start();
```

Am I missing something?

**Comment by shoprememvpfactoryhandover** [comment]
Hi Stephan,

sorry to hear about your problems with 1.9.2!

I investigated your report and there's definitely a problem with how the `ViewVisibilityBuilder` constructs its animation start- and end-actions.
We didn't catch this in our own applications since we have used custom visibility animations for a long time, and the issue only manifests if you never specify your own animation start/end actions.

I've identified and fixed the issue, and will release 1.9.3 asap.

Thanks for the report!

Cheers
David

**Comment by davidganster** [comment]
Hi Stephan,

sorry to hear about your problems with 1.9.2!

I investigated your report and there's definitely a problem with how the ViewVisibilityBuilder constructs its animation start- and end-actions.
We didn't catch this in our own applications since we have used custom visibility animations for a long time, and the issue only manifests if you never specify your own animation start/end actions.

I've identified and fixed the issue, and will release 1.9.3 asap.

Thanks for the report!

Cheers
David
